information/ternary-operators.md

```diff
@@ -0,0 +1,81 @@
+# Ternary Operators
+
+## What is an Ternary Operator?
+
+A ternary operator is some operation operating on 3 inputs. It's a shortcut for an if/else statement, and is also known as a conditional operator.
+
+
+## How to use Ternary Operators
+
+`expression ? expression : expression`
+
+One can use this in multiple ways as it means you don't need to use loads of if/else statements.
+
+An example would be for a command that can take a mention as an argument or no mention at all.
+
+```js
+const user = message.mentions.users.first();
+if(user) {
+    message.channel.send(message.author.displayAvatarURL());
+};
+
+if(!user) {
+    message.channel.send(message.author.displayAvatarURL());
+};
+```
+
+As you can see, it can be rather tedious and can use multiple lines of unnecessary code. A simplified version using Ternary Operators would be:
+```js
+const user = message.mentions.users.first() === undefined ? message.author : message.mentions.users.first();
+message.channel.send(user.displayAvatarURL());
+```
+
+What this does in this current code example:
+
+* `message.mentions.users.first() === undefined` This will check if in the message, there is not a user mention, 
+
+* `message.author` This then sets the variable `user` as the message author (user object),
+
+* `message.mentions.users.first()` This will the set the variable `user` as the first user mention if the original condition is false.
+
+
+In pseudocode:
+
+if there is no user mentions, set the user variable as the message.author, if there is a mention, set the user variable as the user mention.
+
+You can see how it cut down so many lines of code and makes it look neater.
+
+
+## What does it actually do?
+
+* The first operand is implicitly converted to bool. It is evaluated and all side effects are completed before continuing.

```

Comment:
In fact, it's all under [Truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and [Falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy). What is implicitly made is the coercion in the Boolean context.